repository/BaselineOfSeaside3.package/GRPlatform.extension/instance/pharoVersionsFrom..st

```diff
@@ -0,0 +1,5 @@
+*BaselineOfSeaside3
+pharoVersionsFrom: aVersion
+	"Return a list of pharo version from the major version as parameter to the major version of the image"
+
+	^ {}
```

Comment:
That's non-platform specific code. We should stick to standard Smalltalk Syntax here and avoid Squeak extensions.